addons/mrp/models/stock_move.py

```diff
@@ -274,6 +274,12 @@ def _onchange_product_uom_qty(self):
             new_qty = float_round((mo.qty_producing - mo.qty_produced) * self.unit_factor, precision_rounding=self.product_uom.rounding)
             self.quantity = new_qty
 
+    @api.onchange('quantity')
+    def _onchange_quantity(self):
+        if self.raw_material_production_id and not self.manual_consumption and \
+           self.should_consume_qty != self.quantity:
```

Comment:
"Always" use our float utils

Reply:
Using the rounding precision of the `product_uom` of the stock move made at least 10 test fail because they use product without uom 🙃

I added a `precision_rounding = self.product_uom.rounding or 0.01`, If you prefer I can change all the tests..